src/main/java/org/spdx/licenseTemplate/HtmlTemplateOutputHandler.java

```diff
@@ -99,7 +99,7 @@ private void removeEndParagraphTag() {
 	 * @param id ID used for the div 
 	 * @return a replaceable string
 	 */
-	public static String formatReplaceabledHTML(String text, String id) {
```

Comment:
Since this is a public method, we'll need to keep the old miss-spelled method for compatibility.

Ideally, we'll just add the incorrectly spelled method with a single line to call the correctly spelled method and mark it as deprecated.

Reply:
OK, I'll add a new commit.